db/dev_seeds/proposals.rb

```diff
@@ -1,3 +1,26 @@
+IMAGE_FILES = %w{
+  firdouss-ross-414668-unsplash_846x475.jpg
+  nathan-dumlao-496190-unsplash_713x475.jpg
+  steve-harvey-597760-unsplash_713x475.jpg
+  tim-mossholder-302931-unsplash_713x475.jpg
+}.map do |filename|
+  File.new(Rails.root.join("db",
+                           "dev_seeds",
+                           "images",
+                           "proposals", filename))
+end
+
+def add_image_to( imageable )
```

Comment:
Please amend this commit without spaces between the `add_image_to` argument, thanks! :heart: